### PR TITLE
Should be SSL/TLS (SMTPS) or STARTTLS (SMTP + STARTTLS extension)

### PR DESCRIPTION
**Pull request by andrepereiradasilva**
### Description

The option in "SMTP Security" in global config are "SSL" or "TLS" should be "SSL/TLS" (i.e. SMTPS) or "STARTTLS" (i.e. SMTP + STARTTLS extension).

See https://github.com/joomla/joomla-cms/issues/6798
### Testing

Just a language change in global config mail server Security "SMTP Security" field and tooltip.
### B/C

None.
### More info
- SMTP https://tools.ietf.org/html/rfc5321
- SMTP Start TLS https://tools.ietf.org/html/rfc3207
- https://www.fastmail.com/help/technical/ssltlsstarttls.html

Note: Any improvments in the text are welcome.

### CODE REVIEW

File: administrator/language/en-GB/en-GB.com_config.ini [hunk 1]
```diff
@@ -120,7 +120,7 @@ COM_CONFIG_FIELD_MAIL_SMTP_PASSWORD_DESC="Enter the password for the SMTP host."
 COM_CONFIG_FIELD_MAIL_SMTP_PASSWORD_LABEL="SMTP Password"
 COM_CONFIG_FIELD_MAIL_SMTP_PORT_DESC="Enter the port number of your SMTP server. Use 25 for most unsecured servers and 465 for most secure servers."
 COM_CONFIG_FIELD_MAIL_SMTP_PORT_LABEL="SMTP Port"
-COM_CONFIG_FIELD_MAIL_SMTP_SECURE_DESC="Select the security model that your SMTP server uses."
+COM_CONFIG_FIELD_MAIL_SMTP_SECURE_DESC="Select the security model that your SMTP server uses.<br />- None for no encryption<br />- SSL/TLS for SMTPS (normally in port 465)<br />- STARTTLS for SMTP with STARTTLS extension (normally in port 25 or port 587)"
```
Comment (brianteeman) [inline administrator/language/en-GB/en-GB.com_config.ini]: ON port not IN port

**Comment by andrepereiradasilva** [comment]
ok will correct that later.

Also notice that https://github.com/andrepereiradasilva/joomla-cms/blob/staging/administrator/language/en-GB/en-GB.com_config.ini#L121 is not correct, instead of:
> "Enter the port number of your SMTP server. Use 25 for most unsecured servers and 465 for most secure servers."

Port 25 and 587 can be used for secure servers with STARTTLS, and 465 is used for SMTPS. I think it should be something like:
> "Enter the port number of the SMTP server Joomla will use to send e-mails. Usually:<br />- 25 when using a unsecure mail server<br />- 465 when using a secure server with SMTPS<br />- 25 or 587 when using a secure server with SMTP with STARTTLS extension."

**Comment by andrepereiradasilva** [comment]
Done. Also added some other language improvements.
Thanks @brianteeman for the feedback.

File: administrator/language/en-GB/en-GB.com_config.ini [hunk 2]
```diff
@@ -118,9 +118,9 @@ COM_CONFIG_FIELD_MAIL_SMTP_HOST_DESC="Enter the name of the SMTP host."
 COM_CONFIG_FIELD_MAIL_SMTP_HOST_LABEL="SMTP Host"
 COM_CONFIG_FIELD_MAIL_SMTP_PASSWORD_DESC="Enter the password for the SMTP host."
 COM_CONFIG_FIELD_MAIL_SMTP_PASSWORD_LABEL="SMTP Password"
-COM_CONFIG_FIELD_MAIL_SMTP_PORT_DESC="Enter the port number of your SMTP server. Use 25 for most unsecured servers and 465 for most secure servers."
+COM_CONFIG_FIELD_MAIL_SMTP_PORT_DESC="Enter the port number of the SMTP server Joomla will use to send e-mails. Usually:<br />- 25 when using a unsecure mail server<br />- 465 when using a secure server with SMTPS<br />- 25 or 587 when using a secure server with SMTP with STARTTLS extension."
```
Comment (brianteeman) [inline administrator/language/en-GB/en-GB.com_config.ini]: > a unsecure 

should be
> an unsecure

**Comment by alexonbalangue** [comment]
ok, i don't touch the variable php files because missing some translate to conbinate with exactly variable (need to add)

**Comment by brianteeman** [comment]
Please correct to match the en-gb style guide for email https://joomla.github.io/user-interface-text/?user-interface-text/words2watch.md
> email	No hyphen and lower case. Except Email is obviously correct at the start of a sentence! We will be going with email (lower-case and non hyphenated) for consistency.
<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/8520">issues.joomla.org/joomla-cms/8520</a>.</sub>

**Comment by andrepereiradasilva** [comment]
done

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on 613da31b097cd24c0b6cab1673a8d128d500bbaa<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/8520">issues.joomla.org/joomla-cms/8520</a>.</sub>

**Comment by franz-wohlkoenig** [comment]
I have tested this item :white_check_mark: successfully on 613da31b097cd24c0b6cab1673a8d128d500bbaa<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/8520">issues.joomla.org/joomla-cms/8520</a>.</sub>

**Comment by brianteeman** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/8520">issues.joomla.org/joomla-cms/8520</a>.</sub>

**Pull request merged**